cmake/blt-test/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ endif()
 
 include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
 
+if(WIN32 AND ENABLE_SHARED_LIBS)
+    add_definitions("WIN32_SHARED_LIBS")
```

Comment:
This should be an ``add_target_definitions``, right?

Reply:
perhaps, but trying in a way that applies to all things on windows.